ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/decoders/GeneralDecoderTest.kt

```diff
@@ -6,9 +6,10 @@ import com.akuleshov7.ktoml.exceptions.InvalidEnumValueException
 import com.akuleshov7.ktoml.exceptions.MissingRequiredPropertyException
```

Comment:
can we move it to a separate test? DateTimeDecoder test

Reply:
sure, I'll do that